File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/inteligencia/perdidas/dto/InconsistenciaStockDTO.java
package com.farmacia.sanidadbackend.inteligencia.perdidas.dto;

import lombok.Data;

@Data
public class InconsistenciaStockDTO {
    private Long loteDetalleId;
    private Long medicamentoId;
    private String medicamentoNombre;
    private Integer cantidadLote;
    private Integer cantidadUbicaciones;
    private Integer diferencia;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/InconsistenciaStockDTO.kt
package com.sanidad.movil.data.remote.dto

data class InconsistenciaStockDTO(
    val loteDetalleId: Long,
    val medicamentoId: Long,
    val medicamentoNombre: String,
    val cantidadLote: Int,
    val cantidadUbicaciones: Int,
    val diferencia: Int
)